README.md

```diff
@@ -1,149 +1,86 @@
-# Getting started example for Mbed OS
+# Blinky Mbed OS Example
 
-This guide reviews the steps required to get Blinky with the addition of dynamic OS statistics working on an Mbed OS platform. (Note: To see a rendered example you can import into the Arm Online Compiler, please see our [quick start](https://os.mbed.com/docs/mbed-os/latest/quick-start/online-with-the-online-compiler.html#importing-the-code).)
+## Introduction
+The example project contains variants of an application to blink an LED on supported [Mbed boards](https://os.mbed.com/platforms/).
 
-Please install [Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+The project can be built with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/v5.14/tools/index.html). However, this example project specifically refers to the command line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
 
-## Import the example application
-
-From the command-line, import the example:
-
-```
-mbed import mbed-os-example-blinky
-cd mbed-os-example-blinky
-```
-
-### Now compile
-
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the Arm Compiler:
+Depending on the target, the example project can be built with GCC_ARM, ARM or IAR toolchain. Run the command below after installing ARM Mbed CLI to determine which toolchain support your target.
 
-```
-mbed compile -m K64F -t ARM
+```bash
+$ mbed compile -S
 ```
 
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
+Clone this repository on your system and change the current directory to where the project was cloned.
 
-```
-[snip]
+Modifying default Mbed OS configuration parameters can be done at application level using an application configuration file. By default ARM Mbed CLI looks for [`mbed_app.json`](https://os.mbed.com/docs/mbed-os/v5.14/reference/configuration.html), however the configuration file can be named anything. It can be passed to ARM Mbed CLI using the optional argument `--app-config` of the `compile` sub-command. This project comes with configuration files for the different variants and configurations of the blinky application.
 
-Image: ./BUILD/K64F/GCC_ARM/mbed-os-example-blinky.bin
-```
+Select the section for the variant of the application you wish to build:
+* [Blinky Wait application](#blinky_wait)
```

Comment:
Suggestion: Its not really wait application, Can we rename as Blinky wait apps as Blinky Bare Metal(Non-RTOS)  and Blinky multithreaded(RTOS)

Reply:
"Blinky wait" can be built with and without RTOS, so the suggestion does not work.
"Blinky multithreaded (RTOS)" the RTOS part is redundant information as a multithreaded application implies the existence of an OS.

Reply:
why cant make us blinky single-threaded if Blinky wait go with and without RTOS

Reply:
I did not understand your sentence.

Reply:
Please let me know, Are we trying to show the functionality "Wait" API with blinky application if not why can't we name "blinky wait Application" into "blinky single-threaded application"

Reply:
Decision is to remove extra functionality, like multithreads.